src/oop/special_class_method/medium/q3/QuestionText.md

```diff
@@ -0,0 +1,26 @@
+# oop-special_class_method-medium-q3
+
+## Question Text
+
+You want to write a program where you can control the volume of noisy objects. Make a Java class called `Speaker` that
+has an attribute called **volume** which is initialized in the constructor. The class also includes an accessor and a
+mutator for each attribute.
+
+### Speaker
+
+Your `Speaker` class must have the following items
```

Comment:
I think this should be something like "you are provided with the following items" or "class has the following items already" since given the later "You need to add the following items" implies that these ones will be given to the user